data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/parser/S3EventBridgeNotificationParser.java

```diff
@@ -0,0 +1,36 @@
+package org.opensearch.dataprepper.plugins.source.parser;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.opensearch.dataprepper.plugins.source.S3EventBridgeNotification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.sqs.model.Message;
+
+public class S3EventBridgeNotificationParser implements S3NotificationParser {
+    private static final Logger LOG = LoggerFactory.getLogger(S3EventBridgeNotificationParser.class);
+    private static final String DETAIL_KEY = "detail";
+    private static final String BUCKET_KEY = "bucket";
+    private static final String OBJECT_KEY = "object";
+    private static final String RESOURCES_KEY = "resources";
+
+    @Override
+    public ParsedMessage parseMessage(final Message message, final ObjectMapper objectMapper) {
+        try {
+            final JsonNode parsedNode = objectMapper.readTree(message.body());
+            if (isEventBridgeNotification(parsedNode)) {
```

Comment:
Do we need to even check this?

Reply:
Good call let me update the `S3EventBridgeNotification` class to mark some the fields as required.